Wikipedia/Code/WikipediaURLTranslations.swift

```diff
@@ -0,0 +1,93 @@
+
+/*
+ TODO:
+ - re-export the json with all strings lower-cased then update the "commonNamespace" func to force "namespaceString" to lower-case
+ - replace spaces in the json with "_" too - also replace them in "namespaceString"
+*/
+
+struct WikipediaURLTranslations:Codable {
+    private var languagecode:Dictionary<String, WikipediaURLLanguageCodeTranslations> = Dictionary()
+    private static let sharedLookupTable = WikipediaURLTranslations.init(fileName: "wikipedia-namespaces").languagecode
+    private init(fileName : String) {
```

Comment:
Same comment re: style - all of the Swift doc doesn't have a space between the variable name and the `:`.  Also missing a space between the`:` and `Codable` above. Maybe it's time for a Swift linter